### ISSUE TITLE
Feature request: disable auto memory cleaning, if certain app started (exclusion list)

### ISSUE THREAD

**Issue by ellegance** [henrypp/memreduct#90]
Hi, henrypp!
Your memreduct utility is really great, and auto memory cleaning is very nice feature (user can set it on and forget about memory management at all), but sometimes, when heavy application started (for ex. AAA games), automatic memory cleaning causes freezes. Would be nice to have exclusion list of apps, when auto cleaning will be disabled, and enabled again after such apps were closed.

What's your opinion on that? Do you consider it is nice to have?
Will you accept Pull Requests with such functionality?

**Comment by v00d00m4n** [comment]
good idea but for now you can code a simple process monitor that will close memreduct while some exe from steamapp or origin or uplay or whatever path i runnining and will restore memreduct when this process is closed. Few years ago i coded such a simple universal app monitor that does configurable things before and after some specified process started or finished. I have to look for it and upload its sources, maybe it can help.

**Comment by ThIsLinked** [comment]
Is the project closed? No more updates?

**Comment by henrypp** [comment]
@ThIsLinked 

> Is the project closed? No more updates?

Not closed. I release update asap.

**Comment by ThIsLinked** [comment]
> I release update asap.

Good. The last update was 2019, I thought you abandoned it. To spice things up, write a pre-changelog about what you are working on.

**Comment by xWitt** [comment]
This app is really, really good it cleans way better than other memcleaners but I cannot use it without a exclude list, I use steam link regurlarly but if I try to connect to my steam link machine after memreduct cleaned ram it will not connect untill I restart Steam and close memreduct (because when it cleans ram again i wont be able to connect), I saw two other cleaners that have the same problem but they have exclusion lists and if i put steam.exe and steamwebhelper.exe in the list the problem wont occur as it ignores touching Steam while cleaning the rest.

**Comment by ThIsLinked** [comment]
> [@xWitt](https://github.com/henrypp/memreduct/issues/90#issuecomment-1021590816)
I cannot use it without a exclude 

I think you need to create a separate [Issue](https://github.com/henrypp/memreduct/issues/new) for this question.

**Comment by henrypp** [comment]
memreduct do not call cleanup execution on whatever processes, it works globally

**Comment by techgamers2** [comment]
ty for reply i see this feature is not planned... coz globally clearing causes issues in games or related launchers ... no worries i will try something else ... although i like other features of ur app been using for a year useful for low ram system

**Comment by xWitt** [comment]
Thanks for the explanation, I've been using ReduceMemory, it doesn't clean as good as this app but atleast I can use it without affecting streaming apps and my servers.

**Comment by Midawek** [comment]
I found a workaround for this as of 17.05.2025, try doing with "Working Set" disabled in the memory cleaning settings. I had powertoys features turn on with that on but with this off it seems to be working.
